src/Illuminate/Http/Middleware/TrustProxies.php

```diff
@@ -119,16 +119,6 @@ protected function getTrustedHeaderNames()
         };
     }
 
-    /**
-     * Specify that all proxies should be trusted.
-     *
-     * @return void
-     */
-    public static function all()
-    {
-        return static::at('*');
-    }
-
```

Comment:
I am against the removal for Apps, who are still implementing / extending the illuminate part and are calling it.